<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QPushButton" name="data_extraction_tab_btn">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Extraction</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stack_table_tab_btn">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Stack Table</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="plot_tab_btn">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Plot</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>25</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QPushButton" name="input_selection_btn">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Select Input</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="output_selection_btn">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Select Output</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Output_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Output:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="output_path_text">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>375</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stack_widget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>230</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>230</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(141, 211, 211);</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="init_page"/>
         <widget class="QWidget" name="extraction_page">
          <widget class="QPushButton" name="extract_btn">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>180</y>
             <width>71</width>
             <height>23</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 116, 118);</string>
           </property>
           <property name="text">
            <string>Extract</string>
           </property>
          </widget>
          <widget class="QWidget" name="formLayoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>209</width>
             <height>122</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="leftMargin">
             <number>3</number>
            </property>
            <property name="topMargin">
             <number>3</number>
            </property>
            <property name="rightMargin">
             <number>3</number>
            </property>
            <property name="bottomMargin">
             <number>3</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="page2_2">
              <property name="text">
               <string>Extraction Control</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="auto_size_box">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Auto mode</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="size_extractiono_label">
              <property name="text">
               <string>Size extraction</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="stack_page">
          <widget class="QLabel" name="page2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>5</y>
             <width>91</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Stack Control</string>
           </property>
          </widget>
          <widget class="QPushButton" name="noise_stack_btn">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>180</y>
             <width>91</width>
             <height>23</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 116, 118);</string>
           </property>
           <property name="text">
            <string>Noise Stack</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="plot_page">
          <widget class="QWidget" name="verticalLayoutWidget_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>381</width>
             <height>221</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="leftMargin">
             <number>3</number>
            </property>
            <property name="topMargin">
             <number>3</number>
            </property>
            <property name="rightMargin">
             <number>3</number>
            </property>
            <property name="bottomMargin">
             <number>3</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>5</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <property name="leftMargin">
                 <number>5</number>
                </property>
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="rightMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QLabel" name="page3">
                  <property name="text">
                   <string>Plot Control</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="sid_box">
                  <property name="text">
                   <string>Sid</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="sid_f_box">
                  <property name="text">
                   <string>Sid*f</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="svg_box">
                  <property name="text">
                   <string>Svg</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="sid_id2_box">
                  <property name="text">
                   <string>Sid/Id^2</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="svg_norm_box">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Svg*W*L</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QFormLayout" name="formLayout">
                <property name="horizontalSpacing">
                 <number>4</number>
                </property>
                <property name="leftMargin">
                 <number>10</number>
                </property>
                <property name="topMargin">
                 <number>10</number>
                </property>
                <property name="rightMargin">
                 <number>10</number>
                </property>
                <property name="bottomMargin">
                 <number>10</number>
                </property>
                <item row="0" column="0">
                 <widget class="QCheckBox" name="debug_mode_box">
                  <property name="font">
                   <font>
                    <family>MS Shell Dlg 2</family>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <italic>false</italic>
                    <bold>true</bold>
                    <underline>false</underline>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 0, 0);</string>
                  </property>
                  <property name="text">
                   <string>Preview</string>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                  <property name="autoRepeat">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="leftMargin">
               <number>3</number>
              </property>
              <property name="topMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>3</number>
              </property>
              <property name="bottomMargin">
               <number>5</number>
              </property>
              <item>
               <widget class="QPushButton" name="by_site_btn">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 116, 118);</string>
                </property>
                <property name="text">
                 <string>Plot by site</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="med_only_btn">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 116, 118);</string>
                </property>
                <property name="text">
                 <string>Median</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="min_only_btn">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 116, 118);</string>
                </property>
                <property name="text">
                 <string>Min</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="max_only_btn">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 116, 118);</string>
                </property>
                <property name="text">
                 <string>Max</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="Input_label">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Input:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="input_files_listview">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>2000</width>
           <height>1000</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
